schedule/yast/yast2_ncurses_textmode@svirt-xen-hvm.yaml

```diff
@@ -0,0 +1,13 @@
+---
+name:           yast2_ncurses_textmode@svirt-hyperv
```

Comment:
if more than one, let's create a folder `schedule/yast/yast2_ncurses/` wdyt?

Reply:
Can be

Reply:
Done. Updated the jobGroup as well